sites/advertiser/private-aggregation/k-freq-measurement-worklet.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The scale factor is multiplied by the aggregatable value to maximize
+ * the signal-to-noise ratio. See “Noise and scaling” section in the
+ * Aggregation Fundamentals document to learn more:
+ * - https://developer.chrome.com/en/docs/privacy-sandbox/aggregation-fundamentals
+ */
+const SCALE_FACTOR = 65536;
```

Comment:
Here and elsewhere: I think we'll want a lower factor to allow people to try out the different demos (without waiting a day).